docs/useThrottle.md

```diff
@@ -20,48 +20,72 @@ import { useThrottle } from 'rooks';
 
 ## Usage
 
-```jss
-function ThrottleDemo() {
-  const [number, setNumber] = useState(0);
-  const addNumber = () => setNumber(number + 1);
-  const [addNumberThrottled, isReady] = useThrottle(addNumber, 1000);
-  // isReady is a boolean that tells you whether calling addNumberThrottled at that point
+```jsx
+function Demo() {
+  const [text, setText] = useState("");
+  const [throttleValue, setThrottleValue] = useState("");
+  const [throttledFn, isReady] = useThrottle(changeThrottleValue, 1000);
+  // isReady is a boolean that tells you whether calling throttledFn at that point
   // will fire or not.
   // Once the timeout of 1000ms finishes, isReady will become true to indicate that the next time
-  // addNumberThrottled is called it will run right away.
+  // throttledFn is called it will run right away.
+  const ref = useRef(null);
```

Comment:
What does this `ref` do? 🤔

Reply:
It was from previous code, I think we don't need to use a ref here, I will remove it